src/slab.js

```diff
@@ -9,6 +9,23 @@ function getSignature(content) {
   return hmac.digest('hex')
 }
 
+function concat_path(url, path) {
+  let full_path
+
+  // Sanitize path provided
+  if (path.startsWith('/')) {
+    path = path.substring(1)
+  }
```

Comment:
what if it starts with '///' e.g. ?

if you need some url sanitization I would recommend using already built primitives as it's extremely hard to get right

Reply:
not sure it's relevant

Reply:
Since the path-part is provided within the codebase I can safely assume they are correctly formatted.
Thus I would only keep adding the `/` if the URL doesn't have one in the end.

Reply:
got it